### PR TITLE
add mandala/acala/karura chain info

### PR DESCRIPTION
**Pull request by shunjizhan**

### CODE REVIEW

**Review by ChenChunMing1986Loa — APPROVED** [review]

**Review by ligi — APPROVED** [review]

File: _data/chains/eip155-595.json [hunk 1]
```diff
@@ -1,15 +1,21 @@
...
   },
   "infoURL": "https://acala.network",
-  "shortName": "maca",
+  "shortName": "mda",
   "chainId": 595,
-  "networkId": 595
+  "networkId": 595,
+  "explorers": [{
+    "name": "blockscout",
+    "url": "https://blockscout.mandala.acala.network/",
```
Comment (ligi) [inline _data/chains/eip155-595.json:17]:
```suggestion
    "url": "https://blockscout.mandala.acala.network",
```

File: _data/chains/eip155-596.json [hunk 2]
```diff
@@ -12,5 +12,11 @@
   "shortName": "tkar",
   "chainId": 596,
   "networkId": 596,
-  "slip44": 596 
+  "slip44": 596,
+  "explorers": [{
+    "name": "blockscout",
+    "url": "https://blockscout.karura-dev.aca-dev.network/",
```
Comment (ligi) [inline _data/chains/eip155-596.json:18]:
```suggestion
    "url": "https://blockscout.karura-dev.aca-dev.network",
```

File: _data/chains/eip155-597.json [hunk 3]
```diff
@@ -12,5 +12,11 @@
   "shortName": "taca",
   "chainId": 597,
   "networkId": 597,
-  "slip44": 597
+  "slip44": 597,
+  "explorers": [{
+    "name": "blockscout",
+    "url": "https://blockscout.acala-dev.aca-dev.network/",
```
Comment (ligi) [inline _data/chains/eip155-597.json:18]:
```suggestion
    "url": "https://blockscout.acala-dev.aca-dev.network",
```

File: _data/chains/eip155-686.json [hunk 4]
```diff
@@ -12,5 +12,11 @@
   "shortName": "kar",
   "chainId": 686,
   "networkId": 686,
-  "slip44": 686
+  "slip44": 686,
+  "explorers": [{
+    "name": "blockscout",
+    "url": "https://blockscout.karura.network/",
```
Comment (ligi) [inline _data/chains/eip155-686.json:18]:
```suggestion
    "name": "blockscout",
    "url": "https://blockscout.karura.network",
```

**Review by ligi — CHANGES_REQUESTED** [review]

**Comment by gluneau** [comment]
Hey @ligi, are there more changes needed here or is it good to go?

**Comment by shunjizhan** [comment]
@ligi any updates on this? If there is anything need to be changed plz let me know, thanks!

**Comment by gluneau** [comment]
Hello @ligi, this PR seems to be in limbo.

It's ready to merge.

**Review by gluneau — COMMENTED** [review]
Why close this, please merge it @ligi

**Pull request closed without merging**